src/locales/de/menu.ts

```diff
@@ -17,6 +17,7 @@ export const menu: SimpleTranslationEntries = {
   "username": "Benutzername",
   "password": "Passwort",
   "login": "Anmelden",
+  "orUse": "Or use",
```

Comment:
```suggestion
  "orUse": "Oder nutze",
```

(Again. Really depends on the context)

Reply:
![image](https://github.com/pagefaultgames/pokerogue/assets/5632804/d39de5bc-5572-435a-bbe3-16590e86e848)

Reply:
Okay then its fine like i wrote it